api/v1/loadtest_types.go

```diff
@@ -159,9 +126,9 @@ type Driver struct {
 	// +optional
 	Build *Build `json:"build,omitempty"`
 
-	// Run describes the run container, which is the runtime of the driver for
-	// the actual test.
-	Run Run `json:"run"`
+	// Run describes a list of run container, where the first container is the
```

Comment:
I would make sure all three comments to "Run" are the same (plurals and all):

```
// Run describes a list of run containers. The container for the test driver is always
// the first container on the list.
```

I would make the other comments the same, just with client or server instead of driver.